resources/views/pages/tickets/most-reported-games.blade.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+use function Laravel\Folio\{name};
+
+name('tickets.most-reported-games');
+
+?>
+
+@php
+
+use App\Models\Game;
+use App\Models\Ticket;
+
+$ticketedGames = Ticket::unresolved()
+    ->join('Achievements', 'Achievements.ID', '=', 'Ticket.AchievementID')
+    ->select('GameID', DB::raw('count(*) AS TicketCount'))
+    ->groupBy('GameID')
+    ->orderBy('TicketCount', 'DESC')
+    ->take(100)
+    ->get();
+
+@endphp
+
+<x-app-layout pageTitle="Ticket Manager">
```

Comment:
Recommend changing the page title to "Most Reported Games"

Reply:
Yep. I got the header below, but not the HTML title.